DNN Platform/DotNetNuke.Web/Common/RequestScopeServiceProvider.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace DotNetNuke.Web.Common
```

Comment:
Can we get the .NET FOundation Copyright notice here, you can grab it from other files